include/RAJA/util/View.hpp

```diff
@@ -162,6 +162,11 @@ struct MultiView
   layout_type const layout;
   nc_pointer_type data;
 
+  RAJA_INLINE constexpr MultiView()
+      : layout(RAJA::Layout<1>()),
```

Comment:
I feel that this is not general, can the layout not default initialize itself?

Reply:
Fixed!